recipes/xbyak/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "xbyak" %}
+{% set version = "5.661" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/herumi/xbyak/archive/v{{ version }}.tar.gz
+  sha256: fa82134eb5b9526c0cf876334a0cca38b94a0e7da644af35ea66bde22ed6e6d5
+  patches:
+    - patch-Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - make PREFIX=%LIBRARY_PREFIX% install  # [win]    
+    - make PREFIX=$PREFIX install  # [unix]
```

Comment:
I prefer `not win` b/c neither of those are actually Unix, but I'm an old guy :-p